modules/reactor/src/meshgenerators/PolygonConcentricCircleMeshGeneratorBase.C

```diff
@@ -160,6 +160,9 @@ PolygonConcentricCircleMeshGeneratorBase::validParams()
                         "Whether the outward interface boundaries are created.");
   params.addParam<boundary_id_type>(
       "interface_boundary_id_shift", 0, "Integer used to shift interface boundary IDs.");
+  params.addParam<bool>("generate_side_specific_boundaries",
+                        true,
```

Comment:
This is true for the assmbly?

Reply:
Are there more users who want this setting enabled, or disabled? Would a default of `false` cover a wider user base?

Reply:
I d rather it be disabled by default, unless it s needed for building an assembly in the regular user workflow, in which case I would rather it be enabled

Reply:
not seeing too many failures with false as the default
```
test:meshgenerators/patterned_cartesian_mesh_generator.square_boundary ...................... FAILED (EXODIFF)
test:meshgenerators/patterned_hex_mesh_generator.hexagon_boundary ........................... FAILED (EXODIFF)
test:meshgenerators/patterned_cartesian_mesh_generator.square_boundary_no_interface ......... FAILED (EXODIFF)
test:meshgenerators/patterned_hex_mesh_generator.hexagon_boundary_no_interface .............. FAILED (EXODIFF)
test:meshgenerators/patterned_cartesian_peripheral_modifier.sq_pattern ...................... FAILED (EXODIFF)
test:meshgenerators/polygon_concentric_circle_mesh_generator.full ........................... FAILED (EXODIFF)
test:meshgenerators/polygon_concentric_circle_mesh_generator.full_no_interfaces ............. FAILED (EXODIFF)
test:meshgenerators/polygon_concentric_circle_mesh_generator.full_inward_interfaces ......... FAILED (EXODIFF)
test:meshgenerators/peripheral_ring_mesh_generator.err_invalid_boundary_type_2 FAILED (EXPECTED ERROR MISSING)
```

notably the RGMB workflow is fine 

As you prefer

Reply:
If the setting does cause failures sometimes, it'd probably be good to add an error message telling the user exactly how to fix their input files (you may already have this).

Reply:
Sure, I can set the default to false then, Do you prefer me regold these failed test or set the parameter to true in the inputs?

Reply:
no regolds if we can avoid it please